lib/cmd/common/deploy-target.js

```diff
@@ -20,7 +20,6 @@ deployTargets.usage = "\nfhc deploy-target create name=<the name of the deploy t
                     + "\nfhc deploy-target setapptarget <appid> <id of the deploy target> --env=<environment>"
```

Comment:
My impression was that common will disappear after this changes. Any reason why we still have commands in common?